core/entity/closingDepositDetail.py

```diff
@@ -12,5 +12,14 @@ class ClosingDepositDetail(BaseModel):
     updated_time: Optional[datetime] = None
     created_user_id: Optional[int] = None
     updated_user_id: Optional[int] = None
+    class Config():
```

Comment:
You don't need to create small for this one. Since it's just a bridge table's entity.